rskj-core/src/main/java/org/ethereum/rpc/Web3Impl.java

```diff
@@ -1065,6 +1066,30 @@ public boolean personal_lockAccount(String address) {
         return personalModule.lockAccount(address);
     }
 
+    @Override
+    public String eth_estimateGas(CallArguments args) {
+        return eth_estimateGas(args, null);
+    }
+
+    @Override
+    public String eth_estimateGas(CallArguments args, String bnOrId) {
+        Block block = null;
+        if (null == bnOrId) {
```

Comment:
Nit: bnOrId == null

Reply:
Hey Reynold, what do you mean?

Reply:
to switch to: "if (bnOrId == null)" because its easier to read like that

Reply:
Yes, sorry, that was too vague.